index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
```

Comment:
- [ ] There is an issue when applying the styles because the `<h1>` it's showing with the default color. I think the issue is because you are missing the `<link rel="stylesheet" href="<YOUR_STYLESHEET_PATH_HERE>">` tag in the `<head>` section :pushpin: :smiley:
![image](https://user-images.githubusercontent.com/81522435/143044780-9cc8ac53-b730-4749-bb92-63293e53616b.png)
![image](https://user-images.githubusercontent.com/81522435/143044835-77d7f77c-cf7e-406f-8ed5-24c2eb2786c3.png)

Reply:
Yeah. I have forgotten to include the link to the css file. So now I have added and updated the index.html file.